corehq/apps/data_interfaces/templates/data_interfaces/partials/case_rule_criteria.html

```diff
@@ -65,7 +65,12 @@
         <button type="button" class="btn btn-danger" data-bind="click: $parent.removeFilter"><i class="fa fa-close"></i></button>
       </div>
       <div class="col-xs-6 text-center">
-        <label class="control-label">{% trans "AND" %}</label>
+        <!-- ko if: $parent.criteriaOperator() == 'ALL' -->
```

Comment:
cool! It might be worth hiding this text when it is the first / only item: 

```suggestion
        <!-- ko if: $index > 0 && $parent.criteriaOperator() == 'ALL' -->
```

Reply:
Nice, pushed a commit adding this!